pdfminer/pdfdocument.py

```diff
@@ -99,7 +99,7 @@ def load(self, parser):
                 raise PDFNoValidXRef('Unexpected EOF - file corrupted?')
             if not line:
```

Comment:
Can you change this check to `len(line) == 0`, or `line is None or len(line) == 0`

Reply:
Actually @pietermarsman I struggle to see why this line is needed at all.

On line 96 we already do
```
                if not line.strip():
                    continue
```
so I'm pretty sure this condition will never triggered. Do you agree?

Reply:
Yes

Reply:
Cool, I'll remove it and also address your other comment today :+1: